docs/coordinates/inplace.rst

```diff
@@ -0,0 +1,53 @@
+In Place Modification of Coordinates
+====================================
+
+Coordinates are generally considered to be immutable. If you want to create
+another coordinate frame with different data you should use
+`~astropy.coordinates.BaseCoordinateFrame.realize_frame`, this is the safest way
+to change the data in a frame as it creates a new frame with that data.
+
+Due to the fact that this creates a new frame it is relatively slow, and in some
```

Comment:
Found this sentence very hard to understand.  Maybe: "Creating a new frame can be relatively slow, however, particularly for scalar coordinates. Hence some situations may require that data by changed in-place in an already existing frame object.  This modification ..."